pkgs/by-name/go/gomuks-web/package.nix

```
@@ -0,0 +1,68 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGo123Module,
+  buildNpmPackage,
+  stdenv,
+  olm,
+  withGoolm ? false,
+  ...
+}:
+
+let
+  cppStdLib = if stdenv.hostPlatform.isDarwin then "-lc++" else "-lstdc++";
+
+in
+buildGo123Module rec {
```

```suggestion
buildGo123Module (finalAttrs: {
```

Then `version` -> `finalAttrs.version`